feature/my/src/main/java/com/hankki/feature/my/myjogbo/MyJogboRoute.kt

```diff
@@ -0,0 +1,195 @@
+            title = stringResource(R.string.ask_delete_jogbo),
+            textButtonTitle = stringResource(id = R.string.close),
+            buttonTitle = stringResource(id = R.string.do_delete)
+        )
+    }
+
+    Column(
+        modifier = Modifier
+            .padding(paddingValues)
+            .fillMaxSize()
+            .background(White),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        HankkiTopBar(
+            leadingIcon = {
+                Icon(
+                    painter = painterResource(id = com.hankki.core.designsystem.R.drawable.ic_arrow_left),
+                    contentDescription = "Back",
+                    modifier = if (editMode) Modifier
+                        .padding(start = 9.dp)
+                        .size(44.dp)
+                        .noRippleClickable(onClick = resetJogboState)
+                    else Modifier
+                        .padding(start = 9.dp)
+                        .size(44.dp)
+                        .noRippleClickable(onClick = navigateUp)
+                )
+            },
+            content = {
+                Text(
+                    text = stringResource(R.string.my_store_jogbo),
+                    style = HankkiTheme.typography.sub3,
+                    color = Gray900
+                )
+            },
+            trailingIcon = {
+                Text(
+                    text = if (editMode) stringResource(R.string.delete) else stringResource(R.string.edit),
+                    style = HankkiTheme.typography.body1,
+                    color = Gray800,
+                    modifier = if (editMode) Modifier
+                        .padding(vertical = 12.dp, horizontal = 14.dp)
+                        .padding(end = 8.dp)
+                        .noRippleClickable(onClick = {
+                            showDialog.value = true
+                        })
+                    else Modifier
+                        .padding(vertical = 12.dp, horizontal = 14.dp)
+                        .padding(end = 8.dp)
+                        .noRippleClickable(onClick = updateEditMode)
+                )
+            }
+        )
+
+        Spacer(modifier = Modifier.height(18.dp))
+
+        LazyVerticalGrid(
+            modifier = Modifier.padding(horizontal = 22.dp),
+            columns = GridCells.Fixed(2),
+            verticalArrangement = Arrangement.spacedBy(17.dp),
+            horizontalArrangement = Arrangement.spacedBy(16.dp)
+        ) {
+            item {
+                AddJogboItem(isEditMode = editMode)
+            }
+
+            items(jogboItems.size) { index ->
+                val jogbo = jogboItems[index]
+                JogboItem(
+                    modifier = if (editMode)
+                        Modifier.noRippleClickable(onClick = {})
+                    else
+                        Modifier.noRippleClickable(onClick = navigateToJogboDetail),
```

Comment:
```suggestion
                    modifier = Modifier.noRippleClickable(onClick = if(editMode) {} else navigateToJogboDetail),
```
간단한 설정 하나의 경우에는 이렇게 해도 좋아용